lib/inspec/runner.rb

```diff
@@ -182,58 +182,10 @@ def register_rules(ctx)
 
     private
 
-    def block_source_info(block)
```

Comment:
Not sure why we move this to the RSpecRunner. the runner is the right place. By moving this to the RSpec runner, this method is not available to the mock runner anymore.